src/utils/buffer.ts

```diff
@@ -43,7 +42,7 @@ export function intToBytes(value: number, bytes: number): Buffer {
  * @inner
  */
 export function signedIntToBytes(value: number, bytes: number): Buffer {
```

Comment:
You have to keep this method only because nano contract still uses this to parse int arguments, right?

Reply:
Yes!